modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -13,6 +13,263 @@
+** application/gzip (**.gz**)
+** application/x-gzip (**.gzip**)
+** application/x-7z-compressed (**.7z**)
+** application/x-rar-compressed (**.rar**)
+** application/x-tar (**.tar**)
+** application/x-bzip2 (**.bz2**)
+** application/x-bzip (**.bz**)
+** application/x-tgz (**.tgz**)
+
+[discrete]
+==== Last Modified Filter Chip
+Find the most relevant files in no time. This filter enables users to sort files based on the time they were last modified. Whether you're looking for the latest versions or need to access files from a specific time period, this filter streamlines your search process.
+
+* **Modified Filter:**
+
+** today
+** last 7 days
+** last 30 days
+** this year
+** last year
+
+[discrete]
+=== Simplified Shared with me page
+
+We made significant improvements to the "Shared with Me" section to make your file-sharing experience more efficient and user-friendly.
+
+Previously divided into three parts, the "Shared with Me" section has now been consolidated into a single, simplified section. This change provides a more streamlined view of all shared files, making it easier to navigate and manage your shared content.
+
+[discrete]
+==== Auto-Accept Feature for Shares:
+
+In an effort to simplify your workflow, we have impleneted an auto-accept feature for shares which is enabled per default. This means that any files shared with you will automatically appear in your "Shared with Me" section without the need for manual acceptance, saving you time and effort.
+
+[discrete]
+==== Filter by People:
+
+If you remember only the name of the person who shared a file with you, our new filter option comes to the rescue. You can now filter the shares by the name of people, making it easier to find files shared by specific individuals.
+
+[discrete]
+==== Option to Hide Unwanted Shares:
+
+We understand that sometimes you may receive shares that are not relevant or wanted. To address this, we have introduced a feature that allows you to hide such shares. By hiding a share, you can maintain a clear view of the shares that are important to you, ensuring your "Shared with Me" section remains organized and clutter-free.
+
+**Access to Hidden Shares:**
+If you change your mind and wish to view a hidden share, you can easily do so. A dedicated area for hidden shares has been added, allowing you to revisit and manage any shares you have previously hidden.
+
+[discrete]
+==== Enhanced Search Filter:
+
+To help you quickly find specific shares, we have improved the search functionality within the "Shared with Me" section. This enhanced search filter enables you to efficiently locate files based on various criteria.
+
+[discrete]
+==== Sync Feature for Better Control:
+
+Gain control over which shared files are available for sync on your mobile and desktop devices. This feature is especially useful for managing large files. You can now choose to exclude certain files from syncing right from the start.
+
+[discrete]
+=== Shortcuts
```

Comment:
Not clear from the text that this is a file type and shortcuts are available from within your files / spaces / folder trees / however you want to name it.

Reply:
added a sentence to clarify that. hope thats clear now